tomcat/src/main/java/nextstep/jwp/ui/LoginController.java

```diff
@@ -0,0 +1,72 @@
+package nextstep.jwp.ui;
+
+import nextstep.Application;
+import nextstep.jwp.db.InMemoryUserRepository;
+import nextstep.jwp.exception.AuthenticationException;
+import nextstep.jwp.exception.InvalidRequestException;
+import nextstep.jwp.exception.UserNotFoundException;
+import nextstep.jwp.model.User;
+import org.apache.coyote.http11.request.HttpRequest;
+import org.apache.coyote.http11.request.RequestBody;
+import org.apache.coyote.http11.request.Session;
+import org.apache.coyote.http11.response.HttpResponse;
+import org.apache.coyote.http11.response.HttpStatus;
+import org.apache.coyote.http11.response.ResponseEntity;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class LoginController extends AbstractController {
+
+    private static final Logger log = LoggerFactory.getLogger(Application.class);
```

Comment:
```suggestion
public class LoginController extends AbstractController {

    private static final Logger log = LoggerFactory.getLogger(LoginController .class);
```

getLogger 에 매개변수로 전달하는 클래스는 로거의 이름으로 설정되어
로거 설정 파일에서 클래스 별 로깅 레벨을 달리 설정할 수 있게 사용되는 것으로 알고 있어요.
로깅 시점에 텍스트에도 어느 클래스에서 발생한 로깅인지 나타내는데 사용되기도 하고요.

LoginController에서 발생하는 로그니까 LoginController라고 표기되며 텍스트가 출력되어야 하고,
Application 클래스의 로깅 레벨을 변경할 때 LoginController의 로깅 레벨이 영향받지 않아야할 것 같아요.